packages/fx-core/resource/package.nls.json

```diff
@@ -887,5 +887,6 @@
   "depChecker.linuxDepsNotFound": "Cannot find @SupportedPackages. Install @SupportedPackages manually and restart Visual Studio Code.",
   "depChecker.linuxDepsNotFoundHelpLinkMessage": "Cannot find @SupportedPackages.\n\nTeams Toolkit requires these dependencies.",
   "error.ActionNotExist": "Action not found: %s",
-  "error.ComponentNotExist": "Component not found: %s"
+  "error.ComponentNotExist": "Component not found: %s",
+  "error.BadComponent": "Component '%s' lacks of property: %s"
```

Comment:
Nit: how about adding `.` in the end to make it into a complete sentence?

Reply:
Good catch. Will fix in next PR.